rpc/filters.go

```diff
@@ -112,11 +113,38 @@ func (f *Filter) pollForBlocks() error {
 	}
 }
 
+func (f *Filter) pollForTransactions() error {
+	for {
+		if f.stopped {
+			return nil
+		}
+
+		txs, err := f.backend.PendingTransactions()
+		if err != nil {
+			return err
+		}
+
+		for _, tx := range txs {
+			f.hashes = append(f.hashes, tx.Hash)
+		}
+
+		<-time.After(5 * time.Second)
```

Comment:
this seems like a pretty long delay considering the block times 🤔 I would suggest changing it to <1s

Reply:
sure, perhaps this should be configured ?